From dedd6c12cbce3081ca91866f9c41c378264ae55c Mon Sep 17 00:00:00 2001
From: Nigel Whillier <nigel.w@nosun.ca>
Date: Sun, 13 Mar 2022 14:26:12 -0600
Subject: [PATCH] No documentation

---
 Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Makefile b/Makefile
index 4f4ac0c6e..62316b45a 100644
--- a/Makefile
+++ b/Makefile
@@ -19,9 +19,7 @@ makefiles = \
   misc/zsh/local.mk \
   misc/systemd/local.mk \
   misc/launchd/local.mk \
-  misc/upstart/local.mk \
-  doc/manual/local.mk \
-  doc/internal-api/local.mk
+  misc/upstart/local.mk
 endif
 
 ifeq ($(ENABLE_BUILD)_$(ENABLE_TESTS), yes_yes)
